contracts/core/modules/follow/ProfileFollowModule.sol

```diff
@@ -12,40 +12,38 @@ import {IERC721} from '@openzeppelin/contracts/token/ERC721/IERC721.sol';
  * @title ProfileFollowModule
  * @author Lens Protocol
  *
- * @notice This follow module only allows profiles that are not already following in the current revision to follow.
+ * @notice A Lens Profile NFT token-gated follow module with single follow per token validation.
  */
 contract ProfileFollowModule is FollowValidatorFollowModuleBase {
-    mapping(uint256 => mapping(uint256 => mapping(uint256 => bool)))
-        internal _isProfileFollowingByRevisionByProfile;
-
-    mapping(uint256 => uint256) internal _revisionByProfile;
+    /**
+     * Given two profile IDs tells if the former has already been used to follow the latter.
+     */
+    mapping(uint256 => mapping(uint256 => bool)) public isProfileFollowing;
```

Comment:
@Zer0dot Anticipating your thoughts, maybe you would prefer `isFollowingByFollowerProfileIdByFollowedProfieId` or something like that here.

I did it this way because I did the mapping public so can be used as getter like `isProfileFollowing(followerProfileId, followedProfileId)` and in the code looks also clear, plus is only used in one part of the code which is super simple.

Reply:
Sure, I think it's fine!